README.md

```diff
@@ -1,9 +1,190 @@
 # clever_tools
 
-Fun ROS nodes and stuff for CLEVER drone kit, that is not included to the [Raspberry Pi image](https://github.com/CopterExpress/clever/releases).
+Useful ROS nodes and stuff for CLEVER drone kit, that is not included to the [Raspberry Pi image](https://github.com/CopterExpress/clever/releases).
+
+## Installation
+
+Clone repository to the catkin workspace sources:
+
+```bash
+cd ~/catkin_ws/src
+git clone https://github.com/CopterExpress/clever_tools.git
+```
+
+Execute catkin_make in catkin workspace directory:
+
+```bash
+cd ~/catkin_ws
+catkin_make clever_tools
```

Comment:
Are you sure this is gonna work? After `catkin_make` there should be the make target, do we have `clever_tools` target?

Reply:
Yes, because we have CMakeList for it.